src/ContactHub.js

```diff
@@ -85,17 +75,24 @@ const cleanCustomer = (data: APICustomer): Customer => {
         updatedAt: s.updatedAt && new Date(s.updatedAt)
       }));
 
-    /* Strip nulls and empty arrays recursively from `base` */
+    /* Strip nulls and empty arrays recursively */
     const base = { ...data.base, jobs, likes, subscriptions };
-    customer.base = (compact(base): BaseProperties);
+    customer.base = (compact(base));
 
-    // check for data.base is useful only for avoid flow warning
+    // check for data.base is useful only to avoid a Flow warning
     if (data.base && data.base.dob) {
       customer.base.dob = new Date(data.base.dob);
     }
 
   }
 
+  if (data.consents) {
+    customer.consents = (compact(data.consents));
+    if (data.consents && data.consents.disclaimer && data.consents.disclaimer.date) {
+      customer.consents.disclaimer.date = new Date(data.consents.disclaimer.date);
```

Comment:
you are no longer using `formatToDate`. Is it right?

Reply:
Yes, that was one of the issues reported via email (not in the GitHub issue): the field is a datetime in the API, and using `formatToDate` was cutting the time off.